oauth_dropins/reddit.py

```diff
@@ -103,9 +114,22 @@ def get(self):
                          user_agent='oauth-dropin reddit identity checker')
 
     refresh_token = reddit.auth.authorize(code)
-    user_id = str(reddit.user.me())
+    user = reddit.user.me()
+    # a short list of attributes to grab 
+    # looks like calling json_dumps on the object opens some kind of stream
+    # https://github.com/praw-dev/praw/blob/master/praw/models/reddit/redditor.py#L41
```

Comment:
wow, weird! i'm guessing you meant a different line than the one in this link, but still, odd.

Reply:
ah i was linking to that line just for the names of the attributes.  i'll rearrange.